src/main/java/htsjdk/samtools/util/Histogram.java

```diff
@@ -42,40 +35,46 @@
  *
  * @author Tim Fennell
  */
-public class Histogram<K extends Comparable> extends TreeMap<K, Bin> {
+public final class Histogram<K extends Comparable<K>> implements Serializable {
```

Comment:
That would be a bigger breaking change - I'm sure I have a bunch of uses of this class where I don't care about serialization and am using non-serializable key types.